app/models/schedule.rb

```diff
@@ -3,9 +3,8 @@ class Schedule < ActiveRecord::Base
 
   has_many :order_cycles, through: :order_cycle_schedules
   has_many :order_cycle_schedules, dependent: :destroy
-  has_many :coordinators, -> { uniq }, through: :order_cycles
 
-  validates :order_cycles, presence: true
+  has_many :coordinators, -> { uniq }, through: :order_cycles
```

Comment:
I wonder why is GH DIFF failing here?  Is there some special hidden character here? This is not a new line, the validates line was just removed and an empty line was added above this one, right?

Reply:
yes, that's it. I think git is correct here since this was at line 6 and now is at line 7. I'm not saying it's easy, it was hard for my brain too :joy: